src/applications/survivors-benefits/components/FormAlerts/index.jsx

```diff
@@ -151,3 +151,27 @@ export const handleAlertMaxItems = () => (
     />
   </div>
 );
+
+export const VaForm214138Alert = () => (
+  <va-alert status="warning" background-only>
+    <h3 slot="headline">You’ll need to submit VA Form 21-4138</h3>
+    <p>
+      You’ll need to submit a Statement in Support of Claim (VA Form 21-4138)
+      with the name of the person the child is currently living with and the
+      full address of where the child resides.
+    </p>
+    <p>
+      We’ll ask you to upload these documents at the end of this application. Or
+      you can send them to us by mail.
+    </p>
+    <p>
+      <a
```

Comment:
Can you change this to va-link like
```
<span className="vads-u-display--block">
    <va-link
      href="https://www.va.gov/find-forms/about-form-21p-534ez"
      external
      text="Get VA Form 21P-534EZ to download"
    />
  </span>
```
The regular anchor tag was missing in one of the chapter 6 pages for some reason.